### PR TITLE
Small edits

### PR DESCRIPTION
**Pull request by jmlarson1**

### CODE REVIEW

File: docs/data_structures/libE_specs.rst [hunk 1]
```diff
@@ -256,10 +256,10 @@ libEnsemble is primarily customized by setting options within a ``LibeSpecs`` cl
                     many GPUs.
 
                 **gpus_per_group** [int]:
-                    Number of GPUs for each group in the scheduler. This can be used to deal
-                    with scenarios where nodes have different numbers of GPUs. In effect a
+                    Number of GPUs for each group in the scheduler. This can be used when
+                    running on nodes with different numbers of GPUs. In effect a
                     block of this many GPUs will be treated as a virtual node.
-                    By default the GPUs on a node are treated as a group.
+                    By default the GPUs on each node are treated as a group.
```
Comment (jmlarson1) [inline docs/data_structures/libE_specs.rst:262]: @shuds13 The last line might be confusing. Is that saying that even though the nodes might have different numbers of GPUs, the groups will be a different size by default?

**Comment by jmlarson1** [comment]
Can someone help me understand the comment `# parse_args defaults to False` on line 25 of `libensemble/tests/unit_tests/test_ensemble.py`. Which parse_args call is being referred to?

**Comment by jlnav** [comment]
> Can someone help me understand the comment `# parse_args defaults to False` on line 25 of `libensemble/tests/unit_tests/test_ensemble.py`. Which parse_args call is being referred to?

For a bit we've been able to do: `ensemble = Ensemble(parse_args=True)`, which calls `parse_args()` underneath.

**Review by jlnav — APPROVED** [review]

**Comment by jmlarson1** [comment]
> > Can someone help me understand the comment `# parse_args defaults to False` on line 25 of `libensemble/tests/unit_tests/test_ensemble.py`. Which parse_args call is being referred to?
> 
> For a bit we've been able to do: `ensemble = Ensemble(parse_args=True)`, which calls `parse_args()` underneath.

I guess that isn't clear from how I read that line. Can we add a bit to the comment to help?

**Review by shuds13 — APPROVED** [review]

**Pull request merged**